README.md

```diff
@@ -26,9 +26,9 @@
 
 ## 🛠 Built With <a name="built-with"></a>
 
```

Comment:
[OPTIONAL] I think it would be best if you included links to the Behance website, the author's page, and the creative commons license page in your acknowledgment.

For example:

I would like to thank [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), who was the source of inspiration for this project as required by the [Creative Commons License](https://creativecommons.org/licenses/).